src/all/mangadex/assets/i18n/messages_en.properties

```diff
@@ -76,6 +76,8 @@ original_language=Original language
 original_language_filter_chinese=%s (Manhua)
 original_language_filter_japanese=%s (Manga)
 original_language_filter_korean=%s (Manhwa)
+prefer_title_in_source_language=Prefer title in source language
+prefer_title_in_source_language_summary=If there is an alternate title available which matches the language set for the source, it will be used
```

Comment:
It is not clear what "set for the source" means.

Reply:
While 'source' is used internally, I think something else should be used for user-facing strings. Currently it can be confused for the origin language (manga=Japanese, manhwa=Korean). A bit awkward, but I think calling it the extension language would make it unambiguous.

So:
```ini
prefer_title_in_source_language=Prefer extension language
prefer_title_in_source_language_summary=If there is an alternate title available which matches the extension language, it will be used
```

I think the use of the comma could be better, but I don't have a better suggestion.

Reply:
It would be better if it matches. If not it will be unclear why 'source' was used when it's not in the string itself.